cmd/export-distro/main.go

```diff
@@ -2,6 +2,7 @@
 // Copyright (c) 2017
 // Cavium
 // Mainflux
+// Copyright (c) 2019 Intel Corporation
```

Comment:
Please don't add additional copyrights beyond the initial one.

Reply:
we are following the guidelines for copyright headers from this link: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide#Contributor%27sGuide-SubmissionLicenseandCopyrights

Reply:
This is fine actually. It's accepted practice for contributors adding to an existing file to inline their names as shown above.